neural_compressor/adaptor/torch_utils/teq.py

```diff
@@ -41,24 +41,23 @@ def __init__(
         self,
         model,
         weight_config={},
+        absorb_to_layer={},
+        extra_config={},
         example_inputs=None
     ):
         """
         :param model: the model for quantization
         :param weight_config (dict, optional): contains all info required by GPTQ. Defaults to {}.
         :param example_inputs: inputs for trace 
         """
-
         self.model = model
-        self.num_bits = weight_config.get('wbits', 4)
-        self.group_size = weight_config.get('group_size', -1)
-        self.scheme = weight_config.get('sym', False)
-        self.folding = weight_config.get('folding', True)
+        self.weight_config = weight_config
+        self.folding = extra_config.get('folding', True)
         self.example_inputs = example_inputs
         self.device, self.dtype = self._get_device()
```

Comment:
transformers.conv1d has not supported yet

Reply:
transformers.conv1d?

Reply:
yes, some models use this layer. I left several comments in your abandoned pr before, please check.

Reply:
ok, awq support it? or gptq support it?

Reply:
gptq should have support it

Reply:
will enable conv1d when RTN and AWQ support it.